src/nupic/utils/Random.cpp

```diff
@@ -48,9 +46,6 @@ Random::Random(UInt64 seed) {
   // if seed is zero at this point, there is a logic error.
   NTA_CHECK(seed_ != 0);
   gen.seed(seed_); //seed the generator
-  //distribution ranges
-  dist_uint_32 = std::uniform_int_distribution<UInt32>(0, MAX32);
-  dist_real_64 = std::uniform_real_distribution<Real64>(0.0, 1.0);
```

Comment:
Removed distributions (less uniform) and using only RNG(consistent values across platforms) 
as the distributions do NOT have to provide same results with different implementations, unfortunately! 

https://stackoverflow.com/questions/34903356/c11-random-number-distributions-are-not-consistent-across-platforms-what-al